### DISCUSSION TITLE
Lazy load images on the devices page

### DISCUSSION THREAD

**Discussion by mattheys** [Koenkk/zigbee2mqtt#8099] [General]
Hi, my work laptop isn't very fast and whenever I visit the devices page it grinds to a halt loading all 1500 images in at once in Firefox.

Is it possible to update the Markdown template to enable lazy loading of images so they only download when they get near the viewport?

**Comment by mattheys** [comment]
So if that's not possible, is it possible to get a 2nd page without images on? 1144 Requests to load a single page is quite a lot.

**Comment by Koenkk** [comment]
Should be enabled now, thanks for the suggestion.

**Reply by mattheys** [reply]
Thanks, sorry for the late reply. It's made a huge difference for me.
